src/domain/venue/utils.ts

```diff
@@ -56,6 +65,34 @@ export const getVenuePayload = ({
   };
 };
 
+export const hasAmenity = (venue: VenueNode | null | undefined) => {
+  if (!venue) return false;
+  return ([
+    'hasClothingStorage',
+    'hasSnackEatingPlace',
+    'hasToiletNearby',
+    'hasAreaForGroupWork',
+    'hasIndoorPlayingArea',
+    'hasOutdoorPlayingArea',
+    'outdoorActivity',
+  ] as const).some((field) => venue[field]);
+};
+
+export const hasAmenitiesChanged = (
+  a: Partial<VenueDataFields>,
+  b: Partial<VenueDataFields>
+) => {
+  return ([
+    'hasClothingStorage',
+    'hasSnackEatingPlace',
+    'hasToiletNearby',
+    'hasAreaForGroupWork',
+    'hasIndoorPlayingArea',
+    'hasOutdoorPlayingArea',
+    'outdoorActivity',
+  ] as const).some((field) => a[field] !== b[field]);
+};
```

Comment:
Wow.. very good! One last improvement would be to have these amenity fields in a variables as a string array, then use that. Now the same list is repeated at least in 3 places. After that it would be very easy to add more of these amenities

Reply:
Updated!